code/ui/components/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { forwardRef } from 'react';
+import type { ButtonProps } from '../Button/Button';
+import { Button } from '../Button/Button';
+
+export const IconButton = forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
shouldn't this be deprecated too? Or have I missed something in the confusion?

Reply:
Well, I'm not sure. We could depreciate the full component indeed. To add an icon in the toolbar you need to do:
```tsx
<Button padding="small" variant="ghost">
  <Icons name="facehappy" />
</Button>
```
Maybe it's easier if we keep both so they can use it directly using:
```
<IconButton>
  <Icons name="facehappy" />
</IconButton>
```

Reply:
I see your point. I'm okay with keeping it.

Reply:
Add a section on how to place button in toolbar